shopelectro/logic/header.py

```diff
@@ -10,14 +10,14 @@ def menu_qs() -> pages_models.PageQuerySet:
         pages_models.Page.objects.active()
         .filter(
             Q(slug__in=settings.HEADER_LINKS['add'])
-            & ~Q(slug__in=settings.HEADER_LINKS['exclude'])
             | (
                 # @todo #974:30m  Optimize the header menu query.
                 #  Fetch catalog page for the header menu at the same query.
                 #  root category pages.
                 Q(parent=pages_models.CustomPage.objects.filter(slug='catalog'))
                 & Q(related_model_name=models.Category._meta.db_table)
                 & Q(parent=pages_models.CustomPage.objects.filter(slug='catalog'))
```

Comment:
It seems this condition duplicate [that one](https://github.com/fidals/shopelectro/pull/982/files#diff-b62fd94a6e6e9496d6a434a9aad0a060R17)